support-frontend/assets/helpers/tracking/ophan.js

```diff
@@ -82,7 +83,23 @@ type OphanABPayload = {
 
 // ----- Functions ----- //
 
-const trackComponentEvents = (componentEvent: OphanComponentEvent) => ophan.record({ componentEvent });
+const utcNow = () => {
+  const now = new Date();
+  return Date.UTC(
+    now.getFullYear(),
+    now.getMonth(),
+    now.getDate(),
+    now.getHours(),
+    now.getMinutes(),
+    now.getSeconds(),
+    now.getMilliseconds()
+  );
```

Comment:
I've been searching around and I can't seem to find a better way. I think we want to use the `utc` versions of these methods though? e.g `now.getUTCFullYear`

Reply:
you're quite right!

Reply:
`now.getHours()` etc were giving us local time